User-Stories/Compliance-Manager-User-Stories/ComplianceManager.md

```diff
@@ -0,0 +1,46 @@
+#Compliance Manager
+This describes the role of a Compliance Manager. This is a role with suitable legal understanding so that he will be able to interpret the legal situation and assess the circumstances accordingly so that he will be able to judge the consequences from obligations respectively support teams / POs in identifying suitable measures to remain compliant. According to OCv2.0 specification, this role may be staffed by external resources.
+
+##Responsibilities, tasks and role
+The ```primary goal```of the Compliance Manager should be to achieve compliance. He is the advocate of the 3rd party license holders within the company. 
+
+Thus the role will be held ```accountable``` for 
+* Compliance of all open source used  within the products, that received his approval
+* Suitability of the general|all|project specific black/whitelists 
+* Creation, maintenance and governance of the Open Source Policy 
+
+Typically this will result in the following kind of ```tasks```:
+* Generate, publish, promote and maintain the corporate Open Source Policy
+* Operate (organize, manage agenda, etc.) the Open Source Board meetings
+* Provide support to the projects in determining legal obligations
+* Support teams in open sourcing their own components
+* Review compliance applications and provide approvals, respectively suitable guidance
+
+A Compliance Manager will ```escalate```to the Open Source Board or the Senior Management Sponsor.
+
+A strong dependency esists to the toolchain used.
+
+In a real life scenario the role of the Compliance Manager could be mapped to:
+* A Product Manager or a Product Owner with reasonable legal expertise 
```

Comment:
I am not quite sure whether the term "Product Manager" or "Product Owner" are also common in organizations which are not comapies

Reply:
This aims to be a mapping of potential real world job titles to help matching. If we have other roles from other organization types, we might add them here.

Reply:
How about using Project Manager and explaining in the glossary that this could be a "porduct Manager"

Reply:
The line you mentioned (#23) is actually there to do this sort of mapping. The role is called Compliance Manager and line 23 does the mapping to (line 24) a Product Manager or Owner and others. We might add more?